source/activities.js

```diff
@@ -298,6 +298,15 @@ export class Activities {
     return this.postActivity(activity)
   }
 
+  delete (activity) {
+    return this.postActivity({
+      type: 'Delete',
+      actor: this.actor.id,
+      object: typeof activity === 'string' ? activity : activity.id,
+      to: activity.to
+    })
+  }
+
```

Comment:
I think the object of a Delete needs to be not an activity but the object that is to be deleted. 

It's defined as an [inverse of Create](https://www.w3.org/TR/activitystreams-vocabulary/#inverse), which they say shouldn't negate the original activity (i.e. shouldn't erase the Create) but should instead inverse the side-effect of that activity, (i.e. erase the object that was Created)

Were you able to test this against an immers server? I think it should reject it. Validation in activitypub-express has Delete [require](https://github.com/immers-space/activitypub-express/blob/master/net/validators.js#L25) a [local non-activity object](https://github.com/immers-space/activitypub-express/blob/master/net/validators.js#L22)

Reply:
You are right, this was an oversight